backend/app/services/launch_pad.rb

```diff
@@ -37,9 +37,8 @@ def land
   end
 
   def abort(reason)
-    stage = stage_to_land
     if reason == 'refusedconsent'
```

Comment:
Same thing here, instead of silently continuing if you receive an unexpected reason, why not throw an error?

Reply:
yes that's a good idea. I think I'll update to do so.  Thanks!